src/Viewer.js

```diff
@@ -56,15 +56,15 @@ export default class Viewer extends Diagram {
     }
 
     fire (eventName, context) {
-        this.get('eventBus').fire(eventName, context);
+        return this.get('eventBus').fire(eventName, context);
```

Comment:
👍

Reply:
Are you sure that fire returns something usefull? I thought it had no return value.

Reply:
Yes, diagram-js EventBus returns the return value of the last called listener.

Reply:
I've created a wiki for this.
https://github.com/renew-js/renew-lib/wiki/EventBus